appeals/web/src/server/appeals/appeal-details/appellant-case/outcome-incomplete/outcome-incomplete.router.js

```diff
@@ -20,7 +20,6 @@ router
 		validators.validateDueDateFields,
 		validators.validateDueDateValid,
 		validators.validateDueDateInFuture,
-		validators.validateDueDateIsBusinessDay,
```

Comment:
I think this is the only place `validateDueDateIsBusinessDay` is used, if so it should be removed from `outcome-incomplete.validators.js` too